Blocks/MovingBlock36.md

```diff
@@ -1,38 +1,39 @@
 ---
-title: Moving piston block 36
+title: Moving block 36
 description: Mechanics around the moving piston block
 ---
 
-# moving piston block 36
+# Moving piston block 36
+
+Moving_Piston is commonly called a B36 or Block 36. This is because, before The Flattening<!--FIX: What's that?-->, it had the block ID 36. B36 is created by pistons when they push a block. The piston head also converts itself into a B36. B36 is non-solid<!--FIX:Seems to contradict the part where it says it has a hitbox--> and invisible, although it does have a hitbox.
 
-Moving_Piston is commonly called a B36 or Block 36 in the technical community. This is because, before The Flattening, it had the block ID 36.
-B36 is created through pistons. When a piston pushes a set of blocks, it transforms all blocks in that set into B36 while they move. The piston head also converts itself into a B36. It is non-solid and invisible, although it does have a hitbox.
```

Comment:
the flattening could possibly link to the mc wiki for now? https://minecraft.fandom.com/wiki/Java_Edition_1.13/Flattening

"non-solid" is an old and vague term, it's probably meant in the sense of being non-opaque and non-conductive.

Reply:
Linked